app/services/kredits.js

```diff
@@ -149,45 +144,47 @@ export default Service.extend({
   getContributors() {
     return this.get('contributorsContract')
       .then((contract) => contract.contributorsCount())
-      .then(contributorsCount => {
-        debug('[kredits] contributorsCount:', contributorsCount.toNumber());
+      .then((count) => {
+        count = count.toNumber();
+        debug('[kredits] contributors count:', count);
         let contributors = [];
 
-        for(var id = 1; id <= contributorsCount.toNumber(); id++) {
+        for(var id = 1; id <= count; id++) {
           contributors.push(this.getContributorById(id));
         }
 
         return RSVP.all(contributors);
       });
   },
 
+  replaceIpfsHash(data) {
+    let { ipfsHash: digest, hashFunction, hashSize } = data;
+    data.ipfsHash = fromBytes32({ digest, hashFunction, hashSize });
```

Comment:
as ipfsHash gets replaced here, should the hashFunction and hashSize be removed from the data, those only work with the hex value that we replace